### ISSUE TITLE
i have python 11 but it does not working

### ISSUE THREAD

**Issue by azamet90** [bmaltais/kohya_ss#2949]
17:28:19-921461 INFO     Kohya_ss GUI version: v24.1.7
17:28:19-927968 INFO     Python version is 3.11.0 (main, Oct 24 2022, 18:26:48) [MSC v.1933 64 bit (AMD64)]
17:28:19-929968 ERROR    The current version of python (sys.version_info(major=3, minor=11, micro=0,
                         releaselevel='final', serial=0)) is not appropriate to run Kohya_ss GUI
17:28:19-930968 ERROR    The python version needs to be greater or equal to 3.10.9 and less than 3.11.0

**Comment by ObiLeek** [comment]
You'll need to install a Python version that meets the application's requirements, specifically between versions 3.10.9 and 3.11.0 (excluding 3.11.0). So install Python 3.10.9

**Comment by azamet90** [comment]
> You'll need to install a Python version that meets the application's requirements, specifically between versions 3.10.9 and 3.11.0 (excluding 3.11.0). So install Python 3.10.9

i did and also tested after install. if i click to prepare metadata it creates alot of folder with error.

**Comment by bmaltais** [comment]
Yeah... that function if broken... I have not had time to fix it. If someone want to tackle this and submit a PR I will be happy to merge it in.
